src/main/java/com/hyperwallet/clientsdk/Hyperwallet.java

```diff
@@ -20,7 +21,6 @@
 public class Hyperwallet {
 
     public static final String VERSION = "1.4.2";
```

Comment:
noticed that version is still  `VERSION = "1.4.2"` I thought we reserved `1.x.x` as V3 and `2.x.x` as V4 ? or any equivalent?

Reply:
Willian suggested that it should be 2.0.0 
I've updated the files with that (except pom.xml)